Marlin/src/module/planner.cpp

```diff
@@ -1414,7 +1414,7 @@ void Planner::check_axes_activity() {
   }
 
   void Planner::unapply_leveling(float raw[XYZ]) {
-
+    planner.synchronize();
```

Comment:
No need to use `planner.` prefix within a `Planner::` method.

Reply:
Updated